Taskfile.yml

```diff
@@ -109,6 +114,8 @@ tasks:
       - task dev:cli -- composer reinstall danskernesdigitalebibliotek/dpl-design-system danskernesdigitalebibliotek/dpl-react --no-cache
       # Build dev scripts
       - task dev:cli -- $(cd dev-scripts/dpl-react; composer install)
+      # Pull the images (necessary for the first reset)
+      - task dev:pull
```

Comment:
I wonder what the error message you saw was? Other developers have set up the project without running into this issue.